### ISSUE TITLE
postprocessing should happen before verification

### ISSUE THREAD

**Issue by bchurchill** [StanfordPL/stoke#919]
When using the --results option, the verification sometimes fails because of an unsupported instruction that should be preprocessed away first.

**Comment by stefanheule** [comment]
This is not the behavior of the verification at the very end (that's done even without --results).  Shouldn't the two behave the same (either both postprocess before verification, or both do it after)?

**Comment by bchurchill** [comment]
Yeah, both should verify after postprocessing.  My thought is that what
the user sees is what should be verified.

Berkeley

**Comment by stefanheule** [comment]
Okay, can you change the other place, too, then?  Reopening the ticket.

**Comment by bchurchill** [comment]
I should be able to do this on Monday.

**Comment by bchurchill** [comment]
I haven't yet done this, because while the fix is easy, it's a pain to find an example to test this on.  It's not on anyone's critical path, but it should get done sometime.
